docs/core/integration-quality-scale/rules/unique-config-entry.md

```diff
@@ -17,6 +17,8 @@ This can lead to duplicated devices and entities with unique identifiers collidi
 Any discovery flow must also ensure that a config entry is uniquely identifiable, as otherwise, it would discover devices already set up.
 
 To prevent this, we need to ensure that the user can only set up a device or service once.
+Trying to set up a device or service that is already set up should be prevented by the integration.
+If an update to the configuration is required, it should be handled by the reconfiguration flow instead, as it allows us to be more consistent with what gets updated.
```

Comment:
How about this as an alternative:

```suggestion
Configuration flows initiated manually by the user to set up a new entry should not modify or update existing configuration entries. For these purposes, we have our discovery, options, and reconfiguration flows available instead.
```

Reply:
So to set the terms straight, what do we use Configuration flows for? because all flows like discovery, options, reauth and reconfigure flow are in theory configuration flows right?

Because with that in mind, the reconfigure flow is also a configuration flow initiated by the user

Reply:
Yes: Hence, with the intention to set up a new entry being mentioned there. This will also fit the bill for anything that comes down road regarding nuggets, that also cause new entries.